.verdaccio/conf/config.yaml

```diff
@@ -29,14 +29,21 @@ packages:
     access: $all
     publish: $authenticated
     proxy: no-proxy
+#    proxy: npmjs
```

Comment:
Do we want to keep these comments ?

Reply:
I figured it might be helpful since I needed that to run the ng-updates